### ISSUE TITLE
Panic in QueryRowContext when calling a request with an error

### ISSUE THREAD

**Issue by blackhawk-skat** [sijms/go-ora#505]
If a SQL query fails due to an Oracle error, the program panics:
panic: runtime error: invalid memory address or nil pointer dereference
[test_ora_err.go.txt](https://github.com/sijms/go-ora/files/14067104/test_ora_err.go.txt)

**Comment by sijms** [comment]
please use Golang `sql/database` package
this is the code
```golang
package main

import (
	"context"
	"database/sql"
	"fmt"
	_ "github.com/sijms/go-ora/v2"
	"os"
)

func main() {
	db, err := sql.Open("oracle", os.Getenv("DSN"))
	if err != nil {
		fmt.Println("can't open db: ", err)
		return
	}
	defer func() {
		err = db.Close()
		if err != nil {
			fmt.Println("Can't close db: ", err)
		}
	}()
	ctx := context.Background()
	rData := db.QueryRowContext(ctx, `select dummy, min(dummy) from dual where 1 = 1`, nil)
	//rData := conn.QueryRowContext(ctx, `select 1 from dual where 1 = 1`, nil)
	err = rData.Err()
	if err != nil {
		fmt.Println("Query error: ", err)
		return
	}

}
```

this is the result
```bash
Query error:  ORA-00937: not a single-group group function
```

**Comment by sijms** [comment]
I fix it. for direct code without sql/database

**Comment by sijms** [comment]
fixed in v2.8.7
